### PR TITLE
Markdown test 5

### PR DESCRIPTION
**Pull request by rtisma**

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+
+
+
+
+
+
+
+Add a markdown comment here
+Add a markdown code snippet here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: ignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: ignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: help

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @sonatype-lift help

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @sonatype-lift i need some help bro

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @sonatype-lift HELP

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @sonatype-lift hey there i need help please

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @SonatypE-lIfT helP

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @sonatype help

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @muse help

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @somethingsonatype

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @somethingsonatype help

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @somethingmuseeeeeeeeeee help

[the remark this answers is not in the document]
Comment (ghost) [reply file.txt:13]: I’m LiftBot and I automatically analyze new code in code review, and comment when I find potential bugs. I also accept comments as commands. Just `@sonatype-lift` followed by the command: **ignore** to mark as false positive, **unignore** to undo, and **help** to see this message. [Click here](https://github.com/apps/sonatype-lift/installations/new) to add LiftBot to another repo.

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:13]: @ sonatype help

**Pull request closed without merging**